multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -778,6 +786,11 @@ def from_coco(
             use_crowd=use_crowd,
         )
         if not rois:
+            # discard the rows without valid annotation ONLY when data has annotation
+            # add default placeholder to data without annotation for prediction
+            if not num_annotations:
+                d["image"].append(abs_path)
+                d["rois"].append([[-1, -1, -1, -1, 0]])  # TODO: maybe remove this placeholder
```

Comment:
Can you give me some context on why this specific placeholder is required

Reply:
"rois" is the required feature column during training and evaluation but not for prediction. The requirement is defined here regardless of if it's for training, evaluation, or prediction: https://github.com/FANGAreNotGnu/autogluon/blob/master/multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py#L200-L213 . We have the default placeholder added also for dict/list inputs, e.g. https://github.com/FANGAreNotGnu/autogluon/blob/master/multimodal/src/autogluon/multimodal/utils/object_detection.py#L213-L274